### PR TITLE
Start abapGit with package and open the #2160

### PR DESCRIPTION
**Pull request by christianguenter2**
Set parameter 'PACKAGE' to start abapGit for a package #2160

### CODE REVIEW

File: src/ui/zcl_abapgit_services_abapgit.clas.abap [hunk 1]
```diff
@@ -134,4 +143,82 @@ CLASS zcl_abapgit_services_abapgit IMPLEMENTATION.
...
+    DATA: lo_repo          TYPE REF TO zcl_abapgit_repo,
+          lt_r_package     TYPE RANGE OF devclass,
+          ls_r_package     LIKE LINE OF lt_r_package,
+          lt_superpackages TYPE zif_abapgit_sap_package=>ty_devclass_tt,
+          lo_package       TYPE REF TO zif_abapgit_sap_package.
+
+    FIELD-SYMBOLS: <lo_repo>         TYPE LINE OF zif_abapgit_definitions=>ty_repo_ref_tt,
+                   <lv_superpackage> LIKE LINE OF lt_superpackages.
+
+    lo_package = zcl_abapgit_factory=>get_sap_package( iv_package  ).
```
Comment (larshp) [inline src/ui/zcl_abapgit_services_abapgit.clas.abap]: double space before `)`

File: src/ui/zcl_abapgit_services_abapgit.clas.abap [hunk 2]
```diff
@@ -134,4 +143,82 @@ CLASS zcl_abapgit_services_abapgit IMPLEMENTATION.
...
+    " should be found via package $abapgit
+    lt_superpackages = lo_package->list_superpackages( ).
+    LOOP AT lt_superpackages ASSIGNING <lv_superpackage>.
+      ls_r_package-low = <lv_superpackage>.
+      INSERT ls_r_package INTO TABLE lt_r_package.
+    ENDLOOP.
+
+    LOOP AT zcl_abapgit_repo_srv=>get_instance( )->list( ) ASSIGNING <lo_repo>.
+
+      IF <lo_repo>->get_package( ) IN lt_r_package.
```
Comment (larshp) [inline src/ui/zcl_abapgit_services_abapgit.clas.abap]: is this range needed? just test `= iv_package` instead?

Comment (larshp) [reply src/ui/zcl_abapgit_services_abapgit.clas.abap]: plus EXIT the loop

File: src/ui/zcl_abapgit_services_abapgit.clas.abap (hunk 1 above)
Comment (christianguenter2) [reply src/ui/zcl_abapgit_services_abapgit.clas.abap]: fixed

File: src/ui/zcl_abapgit_services_abapgit.clas.abap (hunk 2 above)
Comment (christianguenter2) [reply src/ui/zcl_abapgit_services_abapgit.clas.abap]: The range is used because we can check `iv_package` and superpackages in one go.

Exit is fixed.

**Pull request merged**